client/js/chart/index.js

```diff
@@ -17,16 +17,87 @@ export function createChart(el, rankingHistory, rankingDates) {
   seriesContainer.setAttribute('class', 'series-container');
   svg.append(seriesContainer);
 
-  let index = 0;
-  const colors = ['blue', 'red', 'green', 'purple', 'orange'];
-
-  // get only the first 7 tracks for now
-  for (const history of [...rankingHistory.values()].slice(0, 7)) {
-    const color = colors[index % colors.length];
-    const series = createSeries(color, history);
+  /** @type {SVGGElement[]} */
+  const seriesElements = [];
+  const rankingHistoryEntries = [...rankingHistory];
+
+  rankingHistoryEntries.forEach(([, history], index) => {
+    const hue = index * 15 % 360;
+    const color = `hsl(${hue},50%,50%)`;
+    const series = createSeries(svg, color, history);
     seriesContainer.append(series);
-    index++;
-  }
+    seriesElements.push(series);
+  });
+
+  let activeSeries = null;
+  let activeX = null;
+  let activeY = null;
```

Comment:
can you add a comment for this? Was a bit confused what activeSeries is.

Reply:
changed in #37